_posts/addons/2000-01-01-getting-started-with-docker-image-addon.md

```diff
@@ -86,47 +95,56 @@ Once you've donloaded all the needed images, you must run them on your system.
 
 ### Run the database image
 
-To run pour database image you need to have a folder which will be used to store your database data. Here we will be using `/mount/db/mongo/my-app`.
+To run pour database image you need to have a folder which will be used to
+store your database data. Here we will be using `/tmp/mongodb-data`.
 
 To launch the DB image type:
 
 ```bash
-docke run --name=<container-name> -v /mount/db/mongo/my-app:/var/lib/<database-version> scalingo/<database-type>:<database-version> /mongo <user> <password> <admin_password>
+docke run \
+  --name=<container-name> \
+  -v /tmp/mongodb-data:/var/lib/<database-version> \
+  scalingo/<database-type>:<database-version> \
+  /mongo <user> <password> <admin_password>
 ```
 
-The `user` and `password` can be set to anything. Just keep them for the next step.
-The `container-name` is a name that will be used for your database container.
-If you are nunning mongodb you must pass the `MONGO_WIRED_TIGER_CACHE_SIZE_MB` environment variable.
+The `user` and `password` can be set to anything. Just keep them for the next
+step. The `container-name` is a name that will be used for your database
+container.
 
 In our case:
 
 ```bash
 docke run -d --name=mongo_my-app \
-    -e MONGO_WIRED_TIGER_CACHE_SIZE_MB=250 \
-    -v /mount/db/mongo/my-app:/var/lib/mongodb \
-    scalingo/mongo:3.2.6-2 /mongo my-app password password
+    -v /tmp/mongodb-data:/var/lib/mongodb \
+    scalingo/mongo:3.2.6-2 \
+    /mongo my-app password password
```

Comment:
/mongo my-app password adminpassword ?
To prevent people from seeing it as a typo.